applications/rag/frontend/container/static/script.js

```diff
@@ -48,27 +48,19 @@ function onReady() {
         enableForm(false);
 
         // Collect filter data
-        var filterEnabled = document.getElementById("toggle-filters").checked;
-        var body = JSON.stringify({
+        let data = {
             prompt: prompt,
```

Comment:
@blackzlq I'm adding some prompts to test in here https://github.com/GoogleCloudPlatform/ai-on-gke/pull/511 

Can you suggest some ways to test prompts with DLP/NLP?

Reply:
Yeah, @imreddy13 sends the doc in chat room.

Reply:
who worked with Robert De Niro and name one film they collaborated? with name filter to select the info_type personal_name and de-identity template to replace the content with info_type name. is good for DLP

and any prompt then set the range bar to 100 should return `The responses is deemed inapproriate for display`

`Which movie will show blowing up a building ` with range value 60 should got the `The responses is deemed inapproriate for display` and if we lower the value it should return the original response.